CallingScriptsButton.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
```

Comment:
I was initially thinking one button for loading another scene and one button for initiating a call on the device. And then each of the scene button's props would specify the text and the scene id/route to load when it was clicked. And then call buttons props would specify the name and number that's being called.